File: fetch_playlist.py
import requests

url = "https://saiptvlive.ahmed-bd-org.workers.dev/102.m3u8"

try:
    # allow_redirects=True দিয়ে ৩০২ রিডাইরেক্ট ফলো করা হচ্ছে
    response = requests.get(url, allow_redirects=True, timeout=10)
    
    if response.status_code == 200:
        with open("playlist.m3u8", "wb") as file:
            file.write(response.content)
        print("✅ Python: ডাটা সফলভাবে playlist.m3u8 ফাইলে সেভ হয়েছে!")
    else:
        print(f"❌ স্ট্যাটাস কোড: {response.status_code}")

except Exception as e:
    print(f"❌ এরর ঘটেছে: {e}")

File: fetch_playlist.js
import fs from 'fs';

async function saveData() {
  try {
    const response = await fetch("https://saiptvlive.ahmed-bd-org.workers.dev/102.m3u8", {
      redirect: 'follow'
    });

    if (!response.ok) {
      throw new Error(`HTTP error! Status: ${response.status}`);
    }

    const data = await response.text();
    fs.writeFileSync("playlist.m3u8", data);
    console.log("✅ Node.js: ডাটা সফলভাবে playlist.m3u8 ফাইলে সেভ হয়েছে!");
  } catch (error) {
    console.error("❌ এরর ঘটেছে:", error.message);
  }
}

saveData();
